cmd/osbuild-worker/config.go

```diff
@@ -113,5 +123,9 @@ func parseConfig(file string) (*workerConfig, error) {
 		}
 	}
 
+	if config.OSBuildExecutor.Type != "host" && config.OSBuildExecutor.Type != "aws.ec2" && config.OSBuildExecutor.Type != "qemu.kvm" {
```

Comment:
I'd prefer a `.Contains` here but eh.

Reply:
is that in go 1.19?

Reply:
Introduced in 1.21, otherwise only available in experimental pkgs

Reply:
Oof, ok.